### PR TITLE
extract main.css in webpack dev config

### PR DESCRIPTION
**Pull request by tsubik**

### CODE REVIEW

File: webpack-dev.config.js [hunk 1]
```diff
@@ -19,20 +20,27 @@ const config = {
       { test: /\.svg$/, use: [ { loader: 'svg-sprite-loader' } ] },
       {
         test: /\.scss$/,
-        use: [
-          'style-loader',
-          'css-loader?modules&importLoaders=1&localIdentName=cw__[name]_[local]',
-          'resolve-url-loader',
```
Comment (aabdaab) [inline webpack-dev.config.js]: did you omit `style-loader` && `resolve-url-loader` on purpose?

Comment (tsubik) [reply webpack-dev.config.js]: hmm, I used the construct from "production" webpack file, as wanted to get the same main.css file as after `compile` command.

Comment (aabdaab) [reply webpack-dev.config.js]: I checked and we're not taking advantage of them, at least of the `resolve-url-loader` anywhere and the difference between `style-loader` and `css-loader` is a bit unclear for me. And if it wasn't in the production config then i'd say it's reduntant now

Comment (tsubik) [reply webpack-dev.config.js]: Previous config was taken from styleguide webpack config, so I think it was just a standard one when using this library.

**Comment by tsubik** [comment]
I have to update readme with resolve symlinks info.

**Comment by tsubik** [comment]
Done, merging.

**Pull request merged**